app/src/main/java/org/mozilla/focus/menu/browser/BlockingItemViewHolder.kt

```diff
@@ -75,7 +77,7 @@ internal class BlockingItemViewHolder(
 
             buttonView.context.components.apply {
                 if (isChecked) {
-                    trackingProtectionUseCases.removeException(fragment.tab.id)
+                    fragment.tab.removeFromExceptions(trackingProtectionUseCases)
```

Comment:
This looks like it could be an issue on AC/GV, as we should be able to remove the exception using the `taId`

Reply:
We use the id in Fenix and it works.

Reply:
Ok, I found the reason, it looks like [this GV](https://github.com/mozilla-mobile/android-components/blob/dca90e65d62d69db63c7d9547d36f880ecf10770/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/TrackingProtectionExceptionFileStorage.kt#L106) call doesn't work for private sessions. I see the same issue on Fenix on private tabs

Reply:
Let's put a fix on AC for both